### PR TITLE
Podcast - Episode 29

### PR DESCRIPTION
**Pull request by ewels**

### CODE REVIEW

File: content/podcast/2024/ep29_Nextflow-2023-2024.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+title=Nextflow 2023 - 2024
+episode=29
+description=We recap of some of the biggest features to be added to Nextflow in 2023 and take a look at some of the things coming in 2024.
+date=2024-01-23
+type=podcast
+subtype=Community highlights
```
Comment (vdauwera) [inline content/podcast/2024/ep29_Nextflow-2023-2024.md:6]: I'm surprised you put this in Community highlights, it's more about technical features?

File: content/podcast/2024/ep29_Nextflow-2023-2024.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+title=Nextflow 2023 - 2024
+episode=29
+description=We recap of some of the biggest features to be added to Nextflow in 2023 and take a look at some of the things coming in 2024.
```
Comment (vdauwera) [inline content/podcast/2024/ep29_Nextflow-2023-2024.md:3]: "We recap of some of the biggest features" 

extra 'of' snuck in there

File: content/podcast/2024/ep29_Nextflow-2023-2024.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+We go over some highlights from 2023 and then talk a bit about what’s coming next.
+
+- 2023
+    - Fusion support on Azure Batch, Google Batch, SLURM, LSF
+        - Discussed in [Podcast Episode 14](https://nextflow.io/podcast/2023/ep14_summer_podcast_release_highlights.html)
+    - Spack integration
+        - See [Podcast Episode 12](https://nextflow.io/podcast/2023/ep12_s3_spack_local.html)
+    - Markdown docs, developer docs
+        - Mentioned in [Episode 14](https://nextflow.io/podcast/2023/ep14_summer_podcast_release_highlights.html)
+        - Didn't mention in the recording, but there are [new docs](https://nextflow.io/docs/latest/developer/index.html) aimed at people wanting to develop Nextflow code now too 🎉
```
Comment (vdauwera) [inline content/podcast/2024/ep29_Nextflow-2023-2024.md:29]: nitpitck but the 'now' seems out of place (i would just delete it)

File: content/podcast/2024/ep29_Nextflow-2023-2024.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    - New `nextflow inspect` command
+        - See [Nextflow docs](https://nextflow.io/docs/edge/cli.html#inspect)
+    - Channel "topics"
+        - See [Nextflow docs](https://nextflow.io/docs/edge/channel.html#topic)
+    - AWS Fargate for compute tasks
+        - **Phil got this wrong in the recording**. The head-node-only support in Seqera Platform did come in 2023 (see [blog post](https://seqera.io/blog/seqera-and-aws-fargate/)), but AWS Fargate support for _compute jobs_ did manage to sneak in to the `23.12.0` edge realease, just before Christmas.
```
Comment (vdauwera) [inline content/podcast/2024/ep29_Nextflow-2023-2024.md:35]: a shocking error ;)

File: content/podcast/2024/ep29_Nextflow-2023-2024.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
+title=Nextflow 2023 - 2024
+episode=29
+description=We recap of some of the biggest features to be added to Nextflow in 2023 and take a look at some of the things coming in 2024.
+date=2024-01-23
+type=podcast
+subtype=Community highlights
+youtubeid=T7YpCcqFQpg
+image=img/podcast_ep29.png
+tags=nextflow,opensource,community,nf-core
```
Comment (vdauwera) [inline content/podcast/2024/ep29_Nextflow-2023-2024.md:9]: consider deleting community,nf-core tags?

**Review by vdauwera — COMMENTED** [review]
a few very minor comments

File: content/podcast/2024/ep29_Nextflow-2023-2024.md (hunk 2 above)
Comment (ewels) [reply content/podcast/2024/ep29_Nextflow-2023-2024.md:3]:
```suggestion
description=We recap some of the biggest features to be added to Nextflow in 2023 and take a look at some of the things coming in 2024.
```

File: content/podcast/2024/ep29_Nextflow-2023-2024.md (hunk 5 above)
Comment (ewels) [reply content/podcast/2024/ep29_Nextflow-2023-2024.md:9]: Yup, was just copy-paste and didn't spend much time thinking about this
```suggestion
tags=nextflow,opensource
```

File: content/podcast/2024/ep29_Nextflow-2023-2024.md (hunk 1 above)
Comment (ewels) [reply content/podcast/2024/ep29_Nextflow-2023-2024.md:6]: Just copied it from our 2023 review (where I probably copied it from somewhere else). So far it looks like we've only ever done:

* Interview
* News & Views
* Community highlights (ep 27 only)

I'm not really sure if / where this tag is used, but I think that my original intention back when we rebooted Channels was to use this to differentiate between devrel team and Rob Lalonde.

```suggestion
subtype=subtype=News and Views
```

Comment (ewels) [reply content/podcast/2024/ep29_Nextflow-2023-2024.md:6]: Ah it shows on the listing page. I guess we can use whatever then. Feeling inspired for a new tag?

File: content/podcast/2024/ep29_Nextflow-2023-2024.md (hunk 4 above)
Comment (ewels) [reply content/podcast/2024/ep29_Nextflow-2023-2024.md:35]: 😱

File: content/podcast/2024/ep29_Nextflow-2023-2024.md (hunk 1 above)
Comment (ewels) [reply content/podcast/2024/ep29_Nextflow-2023-2024.md:6]:
```suggestion
subtype=Technical discussion
```

Comment (ewels) [reply content/podcast/2024/ep29_Nextflow-2023-2024.md:6]:
```suggestion
subtype=Tech deep dive
```

Comment (ewels) [reply content/podcast/2024/ep29_Nextflow-2023-2024.md:6]:
```suggestion
subtype=Nextflow nitty-gritty
```

**Comment by ewels** [comment]
Applying and changing so that I can merge and go to bed, hope that's ok @vdauwera ! Can address any other comments post-merge as needed.

**Pull request merged**